src/main/jbake/content/docs/what.adoc

```diff
@@ -0,0 +1,170 @@
+= Why Hawkular?
+Heiko Rupp
+2015-07-03
+:description: Vision for Hawkular
+:jbake-type: page
+:jbake-status: draft
+
+
+== Why Hawkular?
+
+.NOTE: WHY: what is the purpose, why Hawkular
+
+To be the best tool to monitor JBoss and other Java Middleware helping
+you spending less and more enjoyable time on middleware management.
+
+OR:
+
+Hawkular will reduce the effort and time spent on systems management and monitoring
+and make this task more enjoyable.
+
+
+.NOTE: How
+
+This is done through automation of various aspects like resource discovery, root cause analysis,
+predictive analytics and easy to use user interfaces.
+
+
+== The Goals of Hawkular
+
+Hawkular is clearly a project to best monitor and manage JBoss Projects as well as other Java processes.
+
+For managed objects that Hawkular knows about, it will provide a specific, _easy to use_ environment.
```

Comment:
At this point will the audience know what a 'Managed Object' is at this point. There has not been a definition of a managed object yet.